README.md

```diff
@@ -2,7 +2,10 @@
 
 ## Preface
 
-A lot of information has been written about JavaScript and EcmaScript since both the language creation and the standard definition. The intention of this road map is to share with you one of many path you can take to grok the language and specially to point you to the high quality information in the intent to save you from hitting misleading ( when not completely wrong ) documents outside in the wild. Hopefully your walk is much easier than mine!
+A lot of information has been written about JavaScript and EcmaScript since both the language creation time and the standard definition time.
+The intention of this road map is to share with you one of many path you can take to grok the language and specially to point you to the high quality information  (which I won't be ever able to write in a better way) in the intent to save you from hitting misleading ( when not completely wrong ) documents outside in the wild.
```

Comment:
I think it is not correct to use 'point to' in this case:

point you to the high quality information

should be 'draw you to'. And could it be you meant to say instead of 'high quality information', 'highest quality information'